lighthouse-core/index.js

```diff
@@ -62,6 +62,13 @@ module.exports = function(url, flags, configJSON) {
 
     // kick off a lighthouse run
     resolve(Runner.run(driver, {url, flags, config}));
+  })).catch(err => {
+    if (err.message.toLowerCase().includes('multiple tabs')) {
+      log.error('status', err.message);
+      return Promise.reject('');
```

Comment:
reason why is not to throw a runtime error... and wanted to show a better error message so i do it in the catch but perhaps i need to do it in the lighthouse-cli instead? (while writing i think that's a better option)